src/CodeCompliance/Domain/Checks/PrivateApi/Used/ObjectIsInitializedInBusinessModel.php

```diff
@@ -33,6 +34,14 @@ public function getGuideline(): string
         return 'Please avoid usage of %s in %s';
     }
 
+    /**
+     * @return string
+     */
+    public function getDocumentationUrl(): string
+    {
+        return static::DOCUMENTATION_BASE_URL . 'private-api-is-extended.html';
```

Comment:
what about this?
https://docs.spryker.com/docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-is-used-on-the-project-level.html#example-of-code-that-causes-an-upgradability-error

Reply:
+